_includes/footer.html

```diff
@@ -62,6 +62,7 @@ <h4 class="footerHeaderStyle align-center-md"><b>Resources</b></h4>
                                     <a href="https://www.facebook.com/SugarLabs-187845102582/timeline/" class="ion-icon ion-logo-facebook"></a>
                                     <a href="https://github.com/sugarlabs" class="ion-icon ion-logo-github"></a>
                                     <a href="https://twitter.com/sugar_labs" class="ion-icon ion-logo-twitter"></a>
+                                    <a href="https://mastodon.social/@sugar_labs" class="ion-icon ion-logo-mastodon"></a>
```

Comment:
Can you please put the following "rel" code in? It makes us verified on Mastodon. The example they give is this: <a rel="me" href="https://mastodon.social/@sugar_labs">Mastodon</a>, so just add that to what you have here.

So it would probably be like this:
<a rel="me" href="https://mastodon.social/@sugar_labs" class="ion-icon ion-logo-mastodon"></a>

Reply:
Please remove the anchor tags as they are not visible in the preview, for example, href="https://mastodon.social/@sugar_labs" class="ion-icon ion-logo-mastodon
And what should be the value of rel.

Reply:
Sorry, GH overedited my example. It's: ``<a rel="me" href="https://mastodon.social/@sugar_labs" class="ion-icon ion-logo-mastodon"></a>``